examples/pom.xml

```diff
@@ -17,6 +17,7 @@
 	<name>client-java-examples-parent</name>
 
 	<properties>
+        <guava.version>30.0-jre</guava.version>
```

Comment:
nit: fix the indent to align with the line below

Reply:
The samples use Guava (could also be removed later). These files mark them as "safe" so that the build doesn't fail when `com.google.common` is detected.